rubrics/api/src/main/bundle/rubrics_eu.properties

```diff
@@ -19,6 +19,7 @@ save=Gorde
 done=Egina
 total=Guztira
 confirm_remove=Ziur zaude kendu egin nahi duzula?
+confirm_remove_public=Ziur "{}" publikoki partekatutako errubriketatik eta webgunearen jatorritik kendu nahi duzula?
```

Comment:
@pablosanroman can you check this, please?

Reply:
Hi @bgarciaentornos!
Best with a little change: 
+ confirm_remove_public= Ziur kendu nahi duzula errubrika hau, "{}", publikoki partekatutako errubriketatik eta gunetik bertatik?